### ISSUE TITLE
"Can't install to" error on Linux

### ISSUE THREAD

**Issue by j-a-m-l** [zaquestion/lab#317]
The `install.sh` script fails on Linux (Ubuntu 19.04) with:
```
bash: line 2: _can_install: command not found
Can't install to 
```

**Comment by jdillenberger** [comment]
Same thing here with Ubuntu and with Arch Linux.  Probably a problem with Linux in General.

**Comment by zaquestion** [comment]
Yeah happening on mine too now, weird, defs tested it before the bringing in the last update.

**Comment by zaquestion** [comment]
I'm totally confused, probably missing something stupid

Any of you linux guys see what I'm missing?? This used to work and my google fu is failing me.
```
_can_install() {
  if [[ ! -d "$BINDIR" ]]; then
    mkdir -p "$BINDIR" 2> /dev/null
  fi
  [[ -d "$BINDIR" && -w "$BINDIR" ]]
}

if ! _can_install && [[ $EUID != 0 ]]; then
    sudo "$0" "$@"
    exit "$?"
fi

if ! _can_install; then
  echo "Can't install to $BINDIR"
  exit 1
fi
```

**Comment by jdillenberger** [comment]
I can't find a bug in the script either. But I found something else that points in the direction of the problem and explains why the CI doesn't recognize the problem. 

The weird thing is that `bash ./install.sh` works perfectly fine, but `cat ./install.sh | bash` does not. 

I wonder, if this should be the correct behavior for bash, or if that may even be a bug in bash?

**Comment by bmeneguele** [comment]
Fixed in #612

**Comment by j-a-m-l** [comment]
Thanks!
